evo/main_rpe.py

```diff
@@ -90,6 +90,11 @@ def parser() -> argparse.ArgumentParser:
         "--plot_mode", default=SETTINGS.plot_mode_default,
         help="the axes for plot projection",
         choices=["xy", "xz", "yx", "yz", "zx", "zy", "xyz"])
+    output_opts.add_argument(
+        "--plot_in_fct_of_distance",
+        action="store_false",
+        help="plot in function of distance (otherwise, in funcion of time) "
+        "(default: false)")
```

Comment:
You forgot to change also this arg.

Reply:
Ah, sorry, give me a second

Reply:
Ok, done. Tested it for all three x-axis options. Worked for me